python/cudf/cudf/core/frame.py

```diff
@@ -2126,28 +2126,151 @@ def _unaryop(self, op):
         return self.__class__._from_table(Frame(data, self._index))
 
     def isnull(self):
-        """Identify missing values.
+        """
+        Identify missing values.
+
+        Return a boolean same-sized object indicating if
+        the values are NA. NA values, such as ``None`` or ``numpy.NaN``,
```

Comment:
We don't have `None` values in our objects, and our `NaN` values aren't numpy based. Lets update this wording a bit to indicate such.

Reply:
Updated the docs to specify in detail what we identify as missing/null values. Let me know if it looks good?